business_requirement_deliverable_default/models/business_requirement_deliverable_default.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resouce_lines(self):
```

Comment:
@elicoidal As far I read there is no rule for this kind of method that is just auxiliary (because this method is only called by others methods). So for me it is better to have a self descriptive name and naming just `resource` for me is not desirable. @duanyp1991 and @siyuancn  Do you agree with me?

cc @duanyp1991 @siyuancn @elicoidal